CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## Version 1.2.0beta1 // 31.01.2020
```

Comment:
Kleine Anmerkung: Changelog und Versionserhöhung besser nicht innerhalb eines PRs anbieten. Sowas macht man lieber ganz am Ende direkt im Master-Branch, wenn klar ist, welche Anpassungen von allen Beteiligten in ein Release geflossen sind.

Reply:
Ja ist mir bekannt. Ich würde ja auch die 1.2.0 als angestrebtes Ende bezeichnen.
Daher und auch um zu zeigen was der Stand ist hier bereits die Changelog